src/Compilers/Core/Portable/Operations/IConditionalAccessExpression.cs

```diff
@@ -18,7 +18,7 @@ public interface IConditionalAccessExpression : IOperation
         /// </summary>
         IOperation ConditionalValue { get; }
         /// <summary>
-        /// Expresson that is conditionally accessed.
+        /// Expression that is conditionally accessed.
```

Comment:
Fixed by https://github.com/dotnet/roslyn/pull/21624, please revert to not conflict.

Reply:
Reverted